src/Adapter/Image/ImageRetriever.php

```diff
@@ -48,9 +50,12 @@ class ImageRetriever
      */
     private $link;
 
+    private $isMultipleImageFormatFeatureActive;
+
     public function __construct(Link $link)
     {
         $this->link = $link;
+        $this->isMultipleImageFormatFeatureActive = FeatureFlag::isEnabled(FeatureFlagSettings::FEATURE_FLAG_MULTIPLE_IMAGE_FORMAT);
```

Comment:
Use DI for this. Either inject the FeatureFlag repository or make this call in the DI